Data/Sys/GameSettings/GRZ.ini

```diff
@@ -0,0 +1,22 @@
+# GRZJ13 - Medal of Honor: Rising Sun
```

Comment:
Why is this ini file separate from the one above?

Reply:
The game was released under this game ID in Japan for some reason.